templates/stamp/stamp.json

```diff
@@ -0,0 +1,690 @@
+                "count": "[sub(int(variables('networkSettings').mysqlDataNodeCount), 1)]"
+            },
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[variables('mysqlNodeTemplateUrl')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "networkSettings": {
+                        "value": "[variables('networkSettings')]"
+                    },
+                    "apiSettings": {
+                        "value": "[variables('apiSettings')]"
+                    },
+                    "machineSettings": {
+                        "value": "[variables('mysqlMachineSettings')]"
+                    },
+                    "storageSettings": {
+                        "value": "[variables('storageSettings')]"
+                    },
+                    "customSettings": {
+                        "value": {
+                            "machineIndex": "[copyindex(2)]",
+                            "vmScripts": "[variables('mysqlDBInstallerSettings').scriptsToDownload]",
+                            "commandToExecute": "[concat(variables('mysqlDBInstallerSettings').regularNodeInstallCommand, ' -n ', copyindex(2))]",
+                            "machineIpOffset": "[int(variables('networkSettings').mysqlServerIpOffset)]"
+                        }
+                    }
+                }
+            }
+        },
+
+        {
+            "name": "vmss",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "[variables('apiVersion')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[variables('vmssTemplateUrl')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "networkSettings": {
+                        "value": "[variables('networkSettings')]"
+                    },
+                    "apiSettings": {
+                        "value": "[variables('apiSettings')]"
+                    },
+                    "machineSettings": {
+                        "value": "[variables('frontendMachineSettings')]"
+                    },
+                    "storageSettings": {
+                        "value": "[variables('storageSettings')]"
+                    }
+                }
+            }
+        }
+    ],
+    "outputs": {
+        "mongoReplicaSetKey": {
+            "type": "string",
+            "value": "[variables('mongoReplicaSetKey')]"
+        },
+        "mongoReplicaSetName": {
+            "type": "string",
+            "value": "[variables('mongoReplicaSetName')]"
+        },
+        "mongoServerAdminPassword": {
+            "type": "string",
+            "value": "[variables('networkSettings').mongoServerAdminPassword]"
+        }
```

Comment:
Can we remove mongoServerAdminPassword, since we're providing as parameter?
Anything else worth adding - maybe deployment url (nlb to vmss)?